components/heyy/actions/create-contact/create-contact.mjs

```
@@ -0,0 +1,96 @@
+        ...(labels?.length && {
+          labels: labels.map((name) => ({
+            name,
+          })),
+        }),
+        attributes:
+          attributes && Object.entries(attributes)
+            .reduce((acc, [
+              externalId,
+              value,
+            ]) => ([
+              ...acc,
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)